### PR TITLE
Added JEST for unit testing

### PR DESCRIPTION
**Pull request by Kanishk-Anand**
In reference to : https://github.com/thenewboston-developers/Account-Manager/issues/54 

Changes: 
- Added Jest for unit testing, installed the required dependencies as devDependencies

- Deprecated `.babelrc` in favor or `babel.config.js` as recommended by [Babel](https://babeljs.io/docs/en/configuration)

- Added unit test for `utils/format.ts` in `format.test.js`

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/preset-react', '@babel/preset-typescript'],
+  plugins: ['react-hot-loader/babel'],
+};
```
Comment (angle943) [inline babel.config.js:4]: nice

**Comment by angle943** [comment]
Screen Shot 2020-07-20 at 11 00 55 PM

When I run the tests via `npm run test`, it runs beautifully, but when i try running individual tests, i get the above scary error. Any way around this?

**Comment by kanishk-anand-oyo** [comment]
Let me check this @angle943

**Comment by angle943** [comment]
https://github.com/thenewboston-developers/Account-Manager/pull/74

**Pull request closed without merging**